data/meson.build

```diff
@@ -41,18 +41,33 @@ i18n.merge_file(
     install_dir: get_option('datadir') / 'metainfo',
 )
 
-systemd = dependency('systemd')
-systemd_system_unit_dir = systemd.get_pkgconfig_variable('systemdsystemunitdir')
+busctl_path = get_option('busctlpath')
+systemd_system_unit_dir = get_option('systemdsystemunitdir')
 
-install_data(
-    meson.project_name() + '.check-for-firmware-updates.service',
-    install_dir: systemd_system_unit_dir
-)
+if systemd_system_unit_dir != 'no'
+    systemd = dependency('systemd')
+    if systemd_system_unit_dir == ''
+        systemd_system_unit_dir = systemd.get_pkgconfig_variable('systemdsystemunitdir', define_variable: ['rootprefix', get_option('prefix')])
```

Comment:
```suggestion
        systemd_system_unit_dir = systemd.get_variable('systemdsystemunitdir', pkgconfig_define: ['rootprefix', prefix])
```

Reply:
Yes, I will do that when I finish reviewing #77 since there is no meson requirement in master :-)